greenmail-core/src/main/java/com/icegreen/greenmail/smtp/commands/RcptCommand.java

```diff
@@ -27,7 +27,7 @@
  */
 public class RcptCommand
         extends SmtpCommand {
-    static Pattern param = Pattern.compile("RCPT TO:\\s?<([^>]+)>",
+    static Pattern param = Pattern.compile("RCPT TO:\\s?<([^>]+[^\\s]+)>",

```

Comment:
It is necessary because if it gets an unencoded string with cutoff non us-ascii characters, the validation fails when it shouldn't.